less/mixins.less

```
@@ -1 +1,33 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.centered() {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.vert-align() {
+    display: flex;
+    flex-direction: column-reverse;
+}
+
+.layout(@justify, @align) {
+    display: flex;
+    justify-content: @justify;
+    align-items: @align;
+}
+
+.button-style() {
+    .centered();
+    background:@button-bg;
+    color: @white;
+    border: @navigation-border;
+    width: 15rem;
```

I wouldn't necessarily recommend using rem for widths. It can cause problems with your style depending on the user settings. Rem's should really only be used for fonts.
```suggestion
    width: 100%;
```